hubmap/frontend/src/components/home/Search.js

```diff
@@ -131,15 +131,15 @@ class SearchBox extends React.Component {
                     this.searchState = store.getState().searchState;
 
                     // extract what we need for gene/tissue from search results
-                    const results = this.searchState.response !== undefined
+                    let results = this.searchState.response !== undefined
```

Comment:
I believe I asked why these are changed from `const` to `let`?
- `const` is preferred: for example: https://medium.com/tanda-developers/why-do-we-prefer-const-d805b9dc03be
- These changes seem unrelated to this PR, in any case?

Reply:
I did not make these changes in this. are they coming from some other merge (perhaps from develop) ?

Reply:
Not going into the history too deeply, but it looks like it came from a [merge](https://github.com/hubmapconsortium/hubmap-data-portal/blame/c4b263e0f405c2ca71b6e7c92c33aa8ce4b44a29/hubmap/frontend/src/components/home/Search.js#L134). I'm less interested in the history of this than making sure that every change in this PR is moving us in the right direction.